arangodb/hatch.toml

```diff
@@ -0,0 +1,10 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["38"]
+version = ["3.8"]
+
+[envs.default.overrides]
+matrix.version.env-vars = "ARANGODB_VERSION"
```

Comment:
Will this automatically set `ARANGODB_VERSION` to the correct version depending on the `version` in line 5?

Reply:
Exactly!